api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -122,6 +122,13 @@ message ExtAuthz {
   // :ref:`certificate<envoy_api_field_service.auth.v3.AttributeContext.Peer.certificate>`.
   bool include_peer_certificate = 10;
 
+  // Specifies if HTTP status codes for gRPC calls reflect gRPC status
+  // https://github.com/envoyproxy/envoy/issues/11079
+  //
+  // When this field is true, Envoy will send an HTTP response code appropriate to
+  // the response seen
+  bool http_status_for_grpc_deny = 15;
```

Comment:
This field doesn't reflect the comments; are we always reflecting the grpc-status or only on deny?

Reply:
It's currently on-deny, the comments reflect a prior state in the PR before input from @ramaraochavali.  Will align them based on feedback here, thanks for calling that out.

Reply:
We want only on deny as described in the issue - IMO making it for all status is too broad and needed.

Reply:
comment updated, and formatted as per comment above.